brainbeats-backend/Startup.cs

```diff
@@ -35,6 +36,11 @@ public class Startup {
         });
       });
 
+      services.Configure<KestrelServerOptions>(options =>
+      {
+        options.Limits.MaxRequestBodySize = 52428800; // Bytes; maximum size of a request is 50MB
```

Comment:
Any reason why its a 50 MB cap?

Reply:
Lloyd and I decided that 50MB was a fair enough request size for an upper limit for the time being, up from the 30MB default